src/subscribe.ts

```diff
@@ -40,6 +40,30 @@ const postAck = (iframe: HTMLIFrameElement, event: MessageEvent) => {
     }
 };
 
+/**
+ * Post a SessionLock-event to the checkout iframe.
+ */
+export const postSessionLock = (iframe: HTMLIFrameElement) => {
+    if (iframe.contentWindow) {
+        iframe.contentWindow.postMessage(
```

Comment:
would it make sense to expand the post message bodies with a a sid so we can verify in the checkout that the sids match before we accept the message?

Reply:
Added to both messages